plugins/transformations/sequence2array/srcCxx/Sequence2ArrayTransformation.cxx

```diff
@@ -212,11 +190,11 @@ bool Seq2ArrayTransformation::are_type_compatible(
             case TypeKind::SEQUENCE_TYPE:
             case TypeKind::ARRAY_TYPE:
             case TypeKind::UNION_TYPE:
-                is_compatible = is_compatible && are_type_compatible(
+                is_compatible = is_compatible && are_types_compatible(
                         input_array_type.content_type(),
                         output_array_type.content_type());
                 if (!is_compatible) {
-                    std::string error("Error: content of input member <"
+                    std::string error("content of input member <"
                             + input_array_type.name()
                             + ">, not compatible with content of output member <"
                             + output_array_type.name()
```

Comment:
I think the "nested" exception will already contain the name of the two incompatible members. If you wanted to log another error message at this level, then I suggest logging the names of the two incompatible types.

Reply:
actually, that throws an error because the array_type doesn't have a name, not sure why it does even have that function

I've changed the function to, instead of throwing exceptions, return NULL and log a RS error message, then the upper message prints the member names